pagerduty/data_source_pagerduty_license_test.go

```diff
@@ -0,0 +1,120 @@
+package pagerduty
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccDataSourcePagerDutyLicense_Basic(t *testing.T) {
+	// Note that this test may fail if the test account's license name does not
```

Comment:
I suggest you to please put this test behind a test toggle like in [`pagerduty_incident_workflow`](https://github.com/PagerDuty/terraform-provider-pagerduty/blob/master/pagerduty/resource_pagerduty_incident_workflow_test.go), which also would need that you update the [README](https://github.com/PagerDuty/terraform-provider-pagerduty/blob/1035d95106bd4db130fefff2cb48e505ac098a2d/README.md?plain=1#L154) with the new environment variable guarding this tests from failing in case the user executing them doesn't comply with the requirements.

Reply:
@imjaroiswebdev I removed this note because I realized that it is required that every PagerDuty Account should have at least 1 license for `"Full User"`'s since the owner is a `Full User` and so this test should pass for every single Production Account.